lucid/utils.py

```diff
@@ -24,6 +29,9 @@
 HAPPI_GENERAL_SEARCH_KEYS = ('name', 'prefix', 'stand')
 _HAPPI_CACHE = None
 HAPPI_CACHE_UPDATE_PERIOD = 60 * 30
+NO_LOG_EXCEPTIONS = (KeyboardInterrupt, SystemExit)
+LOG_DOMAINS = {".pcdsn", ".slac.stanford.edu"}
+SCREENSHOT_DIR = pathlib.Path(os.environ.get("LUCID_SCREENSHOT_DIR", "/tmp"))
```

Comment:
Should this be a shared data directory, or is `/tmp` good enough for now?

Alternatively, should we set this in the environment and leave this as-is?

Reply:
So the SOP here is:
- see that there was a lucid bug in grafana
- check the host and id
- ssh into that host, cd to /tmp, check the file that matches the id

Is that correct? A shared directory is probably better I think, to reduce the overhead in hunting down the png. I think it's fine to defer this to the environment. Either way, one thing that's very absent in the grafana display is exactly where you would go to find the png- I'm not sure what field would be appropriate for that. It'd be nice to have a line in there that said the equivalent of "png at /tmp/a1234.png" or "png at /cds/group/pcds/pyps/apps/lucid/error_screenshots/a1234.png"

Reply:
Yes, that's correct. And I think you're right - even if verbose, there should be a full "this is where to find the image(s)".

Reply:
Addressed in 70db32c